### PR TITLE
update log output in Systemcmd

### LINKED ISSUE #515: Too many log lines output in Systemcmd.class to look for the error messge quickly.

### PR DESCRIPTION
**Pull request by baixinsui**
fixed #515

### CODE REVIEW

File: modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/deployers/terraform/utils/SystemCmd.java [hunk 1]
```diff
@@ -136,7 +139,9 @@ private void readProcessOutput(Process process, SystemCmdResult systemCmdResult)
                 threadToReadStdErr.submit(() -> readStream(stdErrorReader, contextMap));
 
         while (!stdOutFuture.isDone() || !stdErrFuture.isDone()) {
-            log.debug("Command output and error streams are still being read.");
+            if (0 == System.currentTimeMillis() % 1000) {
```
[the remark this answers is not in the document]
Comment (baixinsui) [reply modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/deployers/terraform/utils/SystemCmd.java:142]: @sonatype-lift ignoreall

**Review by swaroopar — APPROVED** [review]
LGTM

File: modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/deployers/terraform/utils/SystemCmd.java [hunk 2]
```diff
@@ -83,12 +82,16 @@ public SystemCmdResult execute(String cmd, int waitSecond) {
...
         } catch (final InterruptedException ex) {
             log.error("SystemCmd process be interrupted.");
             Thread.currentThread().interrupt();
             systemCmdResult.setCommandSuccessful(false);
+            systemCmdResult.setCommandStdError(ex.getMessage());
         } catch (ExecutionException e) {
-            throw new TerraformExecutorException(e.getMessage());
+            systemCmdResult.setCommandSuccessful(false);
+            systemCmdResult.setCommandStdError(e.getMessage());
+            throw new RuntimeException(e);
```
Comment (swaroopar) [inline modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/deployers/terraform/utils/SystemCmd.java:94]: We must throw TerraformExecutorException here and avoid throwing any java inbuilt exceptions.

**Review by swaroopar — CHANGES_REQUESTED** [review]
one minor comment added.

File: modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/deployers/terraform/utils/SystemCmd.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (iskey) [reply modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/deployers/terraform/utils/SystemCmd.java:143]: If we use `System.currentTimeMillis()` here, the expression may always be false during a calling.

**Review by iskey — CHANGES_REQUESTED** [review]
some comments added.

**Review by swaroopar — APPROVED** [review]
LGTM

**Review by iskey — APPROVED** [review]
LGTM

**Pull request merged**